### PR TITLE
add IE browser to karma config. closes #10 and refactor .gitignore

### LINKED ISSUE #10: Configure karma runner on IE

### PR DESCRIPTION
**Pull request by weblancaster**
PR add IE setup to karma config and some refactoring in .gitignore

### CODE REVIEW

**Comment by caiogondim** [comment]
@weblancaster Thanks for the PR.
To resolve that issue im willing to fire IE from my Mac, like they do here: https://github.com/angular/ci.angularjs.org/tree/master/bin

You had success doing something similar?

**Comment by weblancaster** [comment]
I haven't tried @caiogondim but we can't run IE on Mac.. the options are remote machine or virtual machine.

Not everybody has VM's and I saw you want to run the test in SauceLabs ( #11 ) so that might solve this.

**Comment by caiogondim** [comment]
The idea is to run locally as well, as Angular ppl does.
*And* in SauceLabs.

And your PR is breaking the build on my machine =/
Guess we will have to do a different karma.conf for Windows.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+
+# Users Environment Variables
+.lock-wscript
+
+# custom
 .publish/
-npm-debug.log
+.idea
```
Comment (matthewoates) [inline .gitignore]: You can be more explicit with these paths.

Instead of:
`.idea`

`/.idea`
which will only match at the root of the tree.

Comment (weblancaster) [inline .gitignore]: Recursively is often a better approach since we might have same files/extensions in other paths @matthewoates

**Comment by caiogondim** [comment]
Guys, thanks so much for the work but i will have to refuse the PR since does not make karma run with IE the way i want.
And the way i want is the way people make it work with Angular. Running IE on a Mac through a VM.

Yes, not everybody has a VMWare, but in order to test the lib properly, we have to test it somehow on IE as well.

I will work in IE8 support in the coming days.
Let me know if any of you are interested and lets work together.

**Comment by matthewoates** [comment]
For IE8 I'm guessing a polyfill for the ES6 array functions or just not using them will get you most/all the way there.

I'm hoping to add websocket support on a coming weekend. I'll suggest an API in the appropriate issue.

**Pull request closed without merging**